src/types/Theme.ts

```diff
@@ -44,4 +45,5 @@ export const themeProps: (keyof Theme)[] = [
     'color-header-support-background',
     'color-graph-shadow',
     'color-footer-background',
+    'color-dropdown-card',
```

Comment:
Please, take a look on BaseChain to include the default value for this var there :)

Reply:
I believe this is already defined in BaseChain: https://github.com/KoyNetwork/open-block-explorer/blob/0441262e004cad81bc5250a0e1f4e3c8158e464e/src/config/BaseChain.ts#L28